src/js/SubmissionContainer.jsx

```diff
@@ -57,11 +60,18 @@ var SubmissionContainer = React.createClass({
 
     if(code > 5) editsContainer = <EditsContainer/>
 
-    if(code > 8){
+    if(code > 9){
+      irs = <IRS/>;
+      uploadForm = null;
```

Comment:
I don't think we want `null` here (or below if `code > 10`). At least how I've been thinking about it, we'll just keep adding new components onto the page as the user move through the process. This way they'll know precisely where they are in the process, etc.